src/components/Tooltip/index.js

```diff
@@ -0,0 +1,56 @@
+// @noSnapshot
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import { positionType } from './types'
+import { Bubble, ChildrenContainer, Container } from './styles'
+import Message from './exposedStyles/Message'
+
+class Tooltip extends PureComponent {
+  state = {
+    showTooltip: false,
+  }
+
+  onShowTooltip = () => this.setState({ showTooltip: true })
+
+  onHideTooltip = () => this.setState({ showTooltip: false })
+
+  render() {
+    const { children, message, position } = this.props
+    const { showTooltip } = this.state
+
+    return (
+      <Container onMouseLeave={this.onHideTooltip}>
```

Comment:
So this component doesn't really work on mobile ? What about adding some support onClick for example ? :)

Reply:
It does work on mobile thanks to `onFocus` prop :)

Reply:
Confirmed working on https://deploy-preview-100--atti-components.netlify.com on android/FF

Seems good to me as it is.